rest/config.go

```diff
@@ -55,31 +55,32 @@ const (
-	MaxCouchbaseOverflow           *int                     `json:",omitempty"` // Max # of overflow sockets to open
-	CouchbaseKeepaliveInterval     *int                     `json:",omitempty"` // TCP keep-alive interval between SG and Couchbase server
-	SlowServerCallWarningThreshold *int                     `json:",omitempty"` // Log warnings if database calls take this many ms
-	MaxIncomingConnections         *int                     `json:",omitempty"` // Max # of incoming HTTP connections to accept
-	MaxFileDescriptors             *uint64                  `json:",omitempty"` // Max # of open file descriptors (RLIMIT_NOFILE)
-	CompressResponses              *bool                    `json:",omitempty"` // If false, disables compression of HTTP responses
-	Databases                      DbConfigMap              `json:",omitempty"` // Pre-configured databases, mapped by name
+	Interface                      *string                  `json:",omitempty"`            // Interface to bind REST API to, default ":4984"
+	SSLCert                        *string                  `json:",omitempty"`            // Path to SSL cert file, or nil
+	SSLKey                         *string                  `json:",omitempty"`            // Path to SSL private key file, or nil
+	ServerReadTimeout              *int                     `json:",omitempty"`            // maximum duration.Second before timing out read of the HTTP(S) request
+	ServerWriteTimeout             *int                     `json:",omitempty"`            // maximum duration.Second before timing out write of the HTTP(S) response
+	AdminInterface                 *string                  `json:",omitempty"`            // Interface to bind admin API to, default ":4985"
+	AdminUI                        *string                  `json:",omitempty"`            // Path to Admin HTML page, if omitted uses bundled HTML
+	ProfileInterface               *string                  `json:",omitempty"`            // Interface to bind Go profile API to (no default)
+	ConfigServer                   *string                  `json:",omitempty"`            // URL of config server (for dynamic db discovery)
+	Facebook                       *FacebookConfig          `json:",omitempty"`            // Configuration for Facebook validation
+	Google                         *GoogleConfig            `json:",omitempty"`            // Configuration for Google validation
+	CORS                           *CORSConfig              `json:",omitempty"`            // Configuration for allowing CORS
+	DeprecatedLog                  []string                 `json:"log,omitempty"`         // Log keywords to enable
+	DeprecatedLogFilePath          *string                  `json:"logFilePath,omitempty"` // Path to log file, if missing write to stderr
+	Logging                        *base.LoggingConfigMap   `json:",omitempty"`            // Pre-configured databases, mapped by name
```

Comment:
Good catch, changed to "// Configuration for logging with optional log file rotation"